python_modules/dagster/dagster/core/types/config.py

```diff
@@ -219,7 +219,11 @@ def __init__(self):
             super(_List, self).__init__(
                 key='List.{inner_type}'.format(inner_type=inner_type.key),
                 name=None,
-                description='List of {inner_type}'.format(inner_type=inner_type.name),
+                description=(
```

Comment:
actually use type_name=print_config_type_to_string(self, with_lines=False) to populate this